libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/offline/TileUnpacker.java

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.services.android.navigation.v5.navigation.offline;
+
+import android.os.AsyncTask;
+
+import java.io.File;
+
+public class TileUnpacker {
+
+  public void unpack(File src, String destPath, UnpackUpdateTask.UpdateListener updateListener) {
+    new UnpackerTask().executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR, src.getAbsolutePath(), destPath);
+    new UnpackUpdateTask(updateListener).executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR, src);
```

Comment:
How are `UnpackerTask` and `UnpackUpdateTask` working together here?  Is the `UnpackUpdateTask` watching the same `File` that's being unpacked in `UnpackerTask`?  Would it make sense to consolidate here and ensure the two tasks don't get out of sync for whatever reason?

Reply:
@danesfeder `UnpackUpdateTask` is checking the size of the TAR, so they don't depend on each other at all. I put them in separate async tasks so they would work in parallel and not slow each other down.

Reply:
Oh okay, I thought `UnpackUpdateTask` was updating progress based on the work being done in `UnpackerTask`

Reply:
@danesfeder  it's determining the progress based on how much the TAR has shrunk.